gps_zoe_m8q_driver:
  ros__parameters:
    i2c_bus: 1
    i2c_address: 0x42
    update_rate: 10.0
    frame_id: "gps_link"
